CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 2.28.2  (October 2024)
+
+- [Fix]: Updates react and react-native modules (and axios dependency). Fixes issue #769
```

Comment:
Where is the axios dependency updated? Want to make sure you did not miss it.

Reply:
It's updated in package-lock.json.